actionview/test/template/asset_tag_helper_test.rb

```diff
@@ -565,9 +565,6 @@ class PlaceholderImage
     def blank?; true; end
     def to_s; "no-image-yet.png"; end
   end
-  def test_image_tag_with_blank_placeholder
```

Comment:
It was testing a rudimentary version of the functionality this PR implements: an object with overloaded to_s.